Package.swift

```diff
@@ -28,5 +28,6 @@ let package = Package(
     name: "Log",
     dependencies: [
         .Package(url: "https://github.com/Zewo/Data.git", majorVersion: 0, minor: 4),
+        .Package(url: "https://github.com/VeniceX/Venice.git", majorVersion: 0, minor: 4),
```

Comment:
Am I missing something? It looks to me like the only thing you're using Venice for is getting the current time. There's got to be a better way to do that.

cc @paulofaria @scottbyrns